packages/graphiql/src/index.tsx

```diff
@@ -10,6 +10,7 @@ import { createRoot } from 'react-dom/client';
 import 'regenerator-runtime/runtime.js';
 
 import 'graphiql/graphiql.css';
+import { getConfig } from './config';
```

Comment:
Can you move this import up to the other JS imports?

We group JS imports and CSS imports separately, using a blank line to separate.